### PR TITLE
Backups

### LINKED ISSUE #267: Import/Export backups

### PR DESCRIPTION
**Pull request by karissa**
fixes #267 

Still needs:
- [x] UI work in the settings
- [x] translation for the export confirmation dialog
- [x] listen to events for progress bar and give user feedback when finished for both import and export

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Is it a work in progress?

**Comment by Jikstra** [comment]
LGTM how do we want to tackle the nicer transition between settings screens? Currently it seems like it just upons another dialog on top of the settings one.

**Comment by karissa** [comment]
@Jikstra good question. It works fine, but yeah having some sort of transition and putting them in the same dialog might be nice. Would take some refactoring

**Comment by karissa** [comment]
@ralphtheninja yes it is a WIP, those 3 things still need to be done

File: src/renderer/components/dialogs/index.js [hunk 1]
```diff
@@ -19,14 +19,16 @@ module.exports = {
...
+  if (!opts) opts = {}
   const tx = window.translate
-  const opts = {
+  var defaultOpts = {
     type: 'question',
     message: message,
     buttons: [tx('dialogs.confirmation.no'), tx('dialogs.confirmation.yes')]
   }
-  remote.dialog.showMessageBox(opts, response => {
+  remote.dialog.showMessageBox(Object.assign(defaultOpts, opts, {}), response => {
```
Comment (ralphtheninja) [inline src/renderer/components/dialogs/index.js]: Why the last `{}`?

Comment (ralphtheninja) [reply src/renderer/components/dialogs/index.js]: Did you mean to do `opts || {}`?

Comment (karissa) [reply src/renderer/components/dialogs/index.js]: creates a copy I think although perhaps not necessary? does Object.assign mutate?

Comment (ralphtheninja) [reply src/renderer/components/dialogs/index.js]: Yes it mutates first argument.

**Comment by karissa** [comment]
ready for review @Jikstra @ralphtheninja

**Pull request merged**